src/js/views/patients/shared/actions_views.js

```diff
@@ -22,11 +26,40 @@ const FormButton = View.extend({
   },
 });
 
+const DetailsTooltip = View.extend({
+  template: hbs`{{far "info-circle"}}`,
+  onRender() {
+    const template = hbs`
+      {{#if flowName}}<p class="action-tooltip__flow"><span class="action-tooltip__flow-icon">{{fas "folder"}}</span>{{ flowName }}</p>{{/if}}
+      <p><span class="action-tooltip__action-icon">{{far "file-alt"}}</span><span class="action-tooltip__action-name">{{ name }}</span></p>
+      <p class="action-tooltip__action-details">{{ details }}</p>
+    `;
+
+    const flow = this.model.getFlow();
+
+    new Tooltip({
+      messageHtml: renderTemplate(template, {
+        name: this.model.get('name'),
+        flowName: flow ? flow.get('name') : null,
+        details: this._formatDetails(this.model.get('details')),
+      }),
+      uiView: this,
+      ui: this.$el,
+    });
+  },
+  _formatDetails(details) {
+    if (!details || details.length <= 140) return details;
+
+    return `${ details.slice(0, 140).trimEnd() }...`;
```

Comment:
Let's use `import trim from 'js/utils/formatting/trim';` as `trimEnd` has an alias in Edge and I don't want to deal with it or check if babel auto-fixes it etc.

Reply:
👍 just pushed up so circle ci can run the tests because I don't think mine can handle a video call and cypress at the same time 🤞